src/Scripting/Core/ScriptOptions.cs

```diff
@@ -36,7 +36,6 @@ private static ImmutableArray<MetadataReference> GetDefaultMetadataReferences()
                 return ImmutableArray<MetadataReference>.Empty;
             }
 
-            // Provide similar surface to mscorlib (netstandard 2.0).
             // These references are resolved lazily. Keep in sync with list in core csi.rsp.
```

Comment:
"similar" is the keyword :) 
Let's not change it now since that'd be a product change. We can add more references later.